From: Cole Robinson <crobinso@redhat.com>
Date: Thu, 13 Jul 2017 16:28:46 -0400
Subject: [PATCH] Disable qmp cpu commands (bug #1467599)

Not for upstream. Emergency workaround for this bug right before
the fedora release:

https://bugzilla.redhat.com/show_bug.cgi?id=1467599
---
 monitor.c | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/monitor.c b/monitor.c
index be282ecb80..7a668ee42f 100644
--- a/monitor.c
+++ b/monitor.c
@@ -983,10 +983,12 @@ static void qmp_unregister_commands_hack(void)
 #ifndef TARGET_ARM
     qmp_unregister_command(&qmp_commands, "query-gic-capabilities");
 #endif
-#if !defined(TARGET_S390X) && !defined(TARGET_I386)
+#if 1 || !defined(TARGET_S390X) && !defined(TARGET_I386)
+    /*XXX emergency workaround for https://bugzilla.redhat.com/show_bug.cgi?id=1467599 */
     qmp_unregister_command(&qmp_commands, "query-cpu-model-expansion");
 #endif
-#if !defined(TARGET_S390X)
+#if 1 || !defined(TARGET_S390X)
+    /*XXX emergency workaround for https://bugzilla.redhat.com/show_bug.cgi?id=1467599 */
     qmp_unregister_command(&qmp_commands, "query-cpu-model-baseline");
     qmp_unregister_command(&qmp_commands, "query-cpu-model-comparison");
 #endif
